plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -85,6 +87,12 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 		case configMapResource:
 			return r.authorizeGet(nodeName, configMapVertexType, attrs)
 		case pvcResource:
+			if attrs.GetSubresource() == "status" {
+				if utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) {
+					return r.authorizeStatusUpdate(nodeName, pvcVertexType, attrs)
+				}
+				return authorizer.DecisionNoOpinion, "cannot get status subresource", nil
```

Comment:
Can you drop this? I don't think it's neccessary and it changes the logging even when the feature is disabled. Let authorizeGet handle it for now.

Reply:
sure

Reply:
fixed